build_tools/ci/cpu_comparison/run.py

```diff
@@ -2181,6 +2251,24 @@ def __init__(self):
                 "tile_pipeline": "pack-peel-4-level-tiling",
                 "run_on_target": "npu4",
             },
+            # matmul4d test where the input M/N/K are outer dim values.
+            # The total input values correspond to a standard matmul
+            # from the above test are M:512, N:4096, K:512.
+            {
+                "M": 16,
+                "N": 128,
+                "K": 8,
```

Comment:
I think it would be better to provide total and inner and then derive the outer from it:
```
"M": 512,
"N": 4096,
"K": 512,
"M0": 32
"N0": 32
"K0": 64
```

Reply:
I'd like to address this comment as a follow-up. Need more changes of Matmul4d and its base class.